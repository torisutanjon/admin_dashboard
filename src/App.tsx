import { Flex } from "@chakra-ui/react";
import { SideNav, MainContainer } from "./components";
function App() {
  return (
    <Flex h={"100vh"} w={"100vw"} bg={"myColors.second"} flexDir={"row"}>
      <SideNav />
      <MainContainer />
    </Flex>
  );
}

export default App;
