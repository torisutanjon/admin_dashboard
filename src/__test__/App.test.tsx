import "@testing-library/jest-dom";
import renderer from "react-test-renderer";
import App from "../App";
import { expect, test } from "@jest/globals";

describe("some test", () => {
  test("random truthy test", () => {
    expect(true).toBeTruthy();
  });
  test("Should App renders correctly", () => {
    const tree = renderer.create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
