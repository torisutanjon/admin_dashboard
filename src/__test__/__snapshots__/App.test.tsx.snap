// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`some test Should App renders correctly 1`] = `
<div
  className="css-n8ahee"
>
  <div
    className="css-ekrz1i"
  >
    <div
      className="css-1ppop12"
    />
    <div
      className="css-z184qw"
    >
      <div
        className="css-10smrcn"
      />
      <button
        className="css-1c67act"
        onClick={[Function]}
      >
        <div
          className="css-xn6rf3"
        />
      </button>
      <button
        className="css-1c67act"
        onClick={[Function]}
      >
        <div
          className="css-xn6rf3"
        >
          <img
            alt=""
            className="chakra-image css-0"
            src="images/dashboard-light.png"
          />
        </div>
      </button>
      <button
        className="css-16shs7h"
        onClick={[Function]}
      >
        <div
          className="css-xn6rf3"
        >
          <img
            alt=""
            className="chakra-image css-0"
            src="images/sales-report-light.png"
          />
        </div>
      </button>
      <button
        className="css-pvoqv"
        onClick={[Function]}
      >
        <div
          className="css-xn6rf3"
        >
          <img
            alt=""
            className="chakra-image css-0"
            src="images/e-commerce-dark.png"
          />
        </div>
      </button>
      <button
        className="css-yeee7m"
        onClick={[Function]}
      >
        <div
          className="css-xn6rf3"
        />
      </button>
      <button
        className="css-1c67act"
        onClick={[Function]}
      >
        <div
          className="css-xn6rf3"
        >
          <img
            alt=""
            className="chakra-image css-0"
            src="images/widgets-light.png"
          />
        </div>
      </button>
      <button
        className="css-1c67act"
        onClick={[Function]}
      >
        <div
          className="css-xn6rf3"
        >
          <img
            alt=""
            className="chakra-image css-0"
            src="images/mail_light.png"
          />
        </div>
      </button>
      <button
        className="css-1c67act"
        onClick={[Function]}
      >
        <div
          className="css-xn6rf3"
        >
          <img
            alt=""
            className="chakra-image css-0"
            src="images/tables-light.png"
          />
        </div>
      </button>
      <button
        className="css-1c67act"
        onClick={[Function]}
      >
        <div
          className="css-xn6rf3"
        >
          <img
            alt=""
            className="chakra-image css-0"
            src="images/maps-light.png"
          />
        </div>
      </button>
      <button
        className="css-1c67act"
        onClick={[Function]}
      >
        <div
          className="css-xn6rf3"
        >
          <img
            alt=""
            className="chakra-image css-0"
            src="images/icons-light.png"
          />
        </div>
      </button>
      <button
        className="css-1c67act"
        onClick={[Function]}
      >
        <div
          className="css-xn6rf3"
        >
          <img
            alt=""
            className="chakra-image css-0"
            src="images/more-apps-light.png"
          />
        </div>
      </button>
      <button
        className="css-1c67act"
        onClick={[Function]}
      >
        <div
          className="css-xn6rf3"
        />
      </button>
      <button
        className="css-19ugnri"
        onClick={[Function]}
      >
        <div
          className="css-xn6rf3"
        />
      </button>
      <div
        className="css-rnfhx9"
      />
    </div>
  </div>
  <div
    className="css-5z1h4f"
  >
    <div
      className="css-1lxwves"
    >
      <div
        className="css-1pkfico"
      >
        <div
          className="css-1njsq8g"
        >
          <input
            className="chakra-input css-0"
            onBlur={[Function]}
            onFocus={[Function]}
          />
          <button
            className="chakra-button css-1w7wrcy"
            type="button"
          >
            <div
              className="css-11rkaa"
            >
              <img
                alt=""
                className="chakra-image css-114u7op"
                src="images/search-icon.png"
              />
            </div>
          </button>
        </div>
        <div
          className="css-ejzxkf"
        >
          <button
            className="css-1eq9puc"
          >
            <div
              className="css-gmuwbf"
            >
              <img
                alt=""
                className="chakra-image css-bh4wo8"
                src="images/notification-gear.png"
              />
            </div>
          </button>
          <button
            className="css-1gvjip5"
          >
            <div
              className="css-gmuwbf"
            >
              <img
                alt=""
                className="chakra-image css-bh4wo8"
                src="images/settings-gear.png"
              />
            </div>
          </button>
          <button
            className="css-1pd83fc"
          >
            <img
              alt=""
              className="chakra-image css-uwwqev"
              src="images/profile.png"
            />
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;
